documentation/modules/proc-configuring-kafka-rack-awareness.adoc

```diff
@@ -10,7 +10,8 @@ The `rack` object has one mandatory field named `topologyKey`.
 This key needs to match one of the labels assigned to the Kubernetes cluster nodes.
 The label is used by Kubernetes when scheduling the Kafka broker pods to nodes.
 If the Kubernetes cluster is running on a cloud provider platform, that label should represent the availability zone where the node is running.
-Usually, the nodes are labeled with `failure-domain.beta.kubernetes.io/zone` that can be easily used as the `topologyKey` value.
+Usually, the nodes are labeled with `topology.kubernetes.io/zone` label (or `failure-domain.beta.kubernetes.io/zone` on older Kubernetes versions) that can be easily used as the `topologyKey` value.
```

Comment:
Do we need the 'Usually'?  _Nodes are assigned a `topology.kubernetes.io/zone` label (or `failure-domain.beta.kubernetes.io/zone` on older Kubernetes versions), which can be used as the `topologyKey` value._

Reply:
Well, it is _usually_ - it can happen that the nodes will use different or no label as well. So I think removing it will suggests that it is always the case which it is not.